.halcyon/pre-install-hook

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+source "${HALCYON_DIR}/src.sh"
+
+
+pre_install_hook () {
+	local tag source_dir install_dir data_dir
+	expect_args tag source_dir install_dir data_dir -- "$@"
+
+	local magic_dir
+	magic_dir="${source_dir}/.halcyon/hoogle-index-0.4.2"
+
+	local -a opts_a
+	opts_a=()
+	opts_a+=( --constraints="${magic_dir}/constraints" )
+	opts_a+=( --sandbox-extra-apps="${magic_dir}/sandbox-extra-apps" )
+	opts_a+=( --sandbox-extra-apps-constraints="${magic_dir}/sandbox-extra-apps-constraints" )
+
+	halcyon install "${opts_a[@]}" 'hoogle-index-0.4.2' || return 1
```

Comment:
That’s right. I haven’t used `hoogle-index` myself, so I wasn’t sure what to do with it.